readme_renderer/__main__.py

```diff
@@ -1,19 +1,31 @@
 from __future__ import absolute_import, print_function
 import argparse
-from readme_renderer.rst import render
 import sys
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
     parser = argparse.ArgumentParser(
-        description="Renders a .rst README to HTML",
+        description="Renders a .rst or .md README to HTML",
+    )
+    parser.add_argument("input", help="Input README file", type=argparse.FileType("r"))
+    parser.add_argument(
+        "-o",
+        "--output",
+        help="Output file (default: stdout)",
+        type=argparse.FileType("w"),
+        default="-",
     )
-    parser.add_argument('input', help="Input README file",
-                        type=argparse.FileType('r'))
-    parser.add_argument('-o', '--output', help="Output file (default: stdout)",
-                        type=argparse.FileType('w'), default='-')
     args = parser.parse_args()
 
+    if args.input.name.split(".")[-1].lower() == "md":
```

Comment:
Would it make more sense to use Pathlib's .suffix and .match on this input instead?